include/ccl_constants.h

```diff
@@ -115,7 +115,8 @@
 /**
  * Default relative precision if not otherwise specified
  */
-#define GSL_EPSREL 1E-4
+#define GSL_EPSREL 2.5E-5
```

Comment:
I'm worried about making CCL slow just to pass this test. Could we make this a tunable parameter through the params.ini file? I'm happy to implement this for you.